README.rst

```diff
@@ -49,17 +49,17 @@ please take a look at our
 Installation
 ------------
 
-coala can be installed with ``pip install coala``. If you need more
-information about the installation and dependencies, take a look at our
-`installation documentation
+To install coala with the official set of analysis routines with a simple
+``pip install coala-bears``.
+
+coala only, without the official bears, can be installed witt
+``pip install coala``. If you need more information about the installation and
+dependencies, take a look at our `installation documentation
 <http://coala.rtfd.org/en/latest/Users/Install.html>`__.
```

Comment:
No - that's only markdown. This is RST
http://docutils.sourceforge.net/docs/user/rst/quickref.html#hyperlink-targets

On Tue, Mar 22, 2016 at 7:48 PM, Adhityaa Chandrasekar <
notifications@github.com> wrote:

> In README.rst
> <https://github.com/coala-analyzer/coala/pull/1943#discussion_r56993029>:
>
> > @@ -49,17 +49,17 @@ please take a look at our
> >  Installation
> >  ------------
> >
> > -coala can be installed with ``pip install coala``. If you need more
> > -information about the installation and dependencies, take a look at our
> > -`installation documentation
> > +To install coala with the official set of analysis routines with a simple
> > +``pip install coala-bears``.
> > +
> > +coala only, without the official bears, can be installed witt
> > +``pip install coala``. If you need more information about the installation and
> > +dependencies, take a look at our `installation documentation
> >  <http://coala.rtfd.org/en/latest/Users/Install.html>`__.
>
> Also wouldn't [description](http://link) work as well?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/coala-analyzer/coala/pull/1943/files/9f3e80aff051e68c368d47f2f9fd3c99c397a652..03a853a11246c97b3d6c5b6b31dd930a97c624a2#r56993029>
>